query-engine/connector-test-kit-rs/query-engine-tests/tests/new/interactive_tx.rs

```diff
@@ -174,7 +174,7 @@ mod interactive_tx {
         ];
 
         // Tx flag is not set, but it executes on an ITX.
-        runner.batch(queries, false).await?;
+        runner.batch(queries, false, None).await?;
```

Comment:
What would happen if a user specifies a different isolation level in a batch than what the itx is running?

Reply:
Do we stop a user from selecting an isolation level in a batch when it is run in an itx?

Reply:
In client, it's not possible: `$transaction` method is not exposed on transaction proxy.
In engine: let me check. I guess we should throw an error?

Reply:
I think it should be fine given you cannot hit that path via the client.